website/pages/en/network/delegating.mdx

```diff
@@ -39,25 +41,25 @@ One thing to consider as well is choosing an Indexer wisely. If you choose an In
 
 ### Choosing a trustworthy Indexer with a fair reward payout for Delegators
 
-This is an important part to understand. First let's discuss three very important values, which are the Delegation Parameters.
+This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
```

Comment:
This is slightly better than before, but there is still something strange about the wording here.

Reply:
```suggestion
This is an important aspect to understand. First, let's discuss three very important values: Delegation Parameters.
```

Reply:
How do we feel about this instead? 
Also, this is the only time I hear the term "Delegating Parameters," so I'm curious if it's supposed to be a proper noun.

Reply:
Good question for @bermchain